### PR TITLE
Finnish

### PR DESCRIPTION
**Pull request by rpinola**

### CODE REVIEW

File: translations/monero-core_fi.ts [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-<TS version="2.1" language="fi" sourcelanguage="en">
+<TS version="2.0" language="fi" sourcelanguage="en">
 <context>
     <name>AddressBook</name>
     <message>
         <location filename="../pages/AddressBook.qml" line="64"/>
         <source>Address</source>
-        <translation>Osoite</translation>
+        <translation>Osoite
+</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: I don't know what happened here but the format got messed up

File: translations/monero-core_fi.ts [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
         <location filename="../pages/AddressBook.qml" line="51"/>
         <source>Qr Code</source>
-        <translation type="unfinished"></translation>
+        <translation>QR Koodi</translation>
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="73"/>
         <source>Payment ID &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</source>
-        <translation>Maksutunniste &lt;font size=&apos;2&apos;&gt;(Valinnainen)&lt;/font&gt;</translation>
+        <translation>Maksu ID</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: This translation is wrong. Please keep it consistent with the source file. I guess it should be something like:
`Maksu ID &lt;font size=&apos;2&apos;&gt;(Valinnainen)&lt;/font&gt;`

File: translations/monero-core_fi.ts [hunk 3]
```diff
@@ -31,12 +32,12 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="82"/>
         <source>Description &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</source>
-        <translation>Kuvaus &lt;font size=&apos;2&apos;&gt;(Valinnainen)&lt;/font&gt;</translation>
+        <translation>Kuvaus &lt;fonttikoko=&apos;2&apos;&gt;(Valinnainen)&lt;/fontti&gt;</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: HTML doesn't have to be translated

File: translations/monero-core_fi.ts [hunk 4]
```diff
@@ -69,46 +70,46 @@
...
         <location filename="../components/DaemonManagerDialog.qml" line="93"/>
         <source>Starting local node in %1 seconds</source>
-        <translation type="unfinished"></translation>
+        <translation>Käynnistetään paikallinen verkkosolmu %1 sekunnissa</translation>
     </message>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="111"/>
         <source>Start daemon (%1)</source>
-        <translation>Käynnistä taustaohjelma (%1)</translation>
+        <translation type="unfinished">Käynnistä daemon(%1)</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: add a space between `daemon` and `(%1)`

File: translations/monero-core_fi.ts [hunk 5]
```diff
@@ -1279,100 +1287,100 @@ The old wallet cache file will be renamed and can be restored later.
...
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="134"/>
         <source>This sets which outputs are known to be spent, and thus not to be used as privacy placeholders in ring signatures. </source>
-        <translation type="unfinished"></translation>
+        <translation>Tämä asettaa mitkä lähdöt ovat tiedetty kulutetun, ja joita ei siten voi käyttää yksityisyyteen rengasallekirjoituksissa</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: please add `.` + whitespace at the end of the string

File: translations/monero-core_fi.ts [hunk 6]
```diff
@@ -1279,100 +1287,100 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="163"/>
         <source>Path to file</source>
-        <translation type="unfinished"></translation>
+        <translation>Polku tiedostoon</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="165"/>
         <source>Filename with outputs to blackball</source>
-        <translation type="unfinished"></translation>
+        <translation>Tiedoston nimi jossa lähdöt:</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: please remove `:`

File: translations/monero-core_fi.ts [hunk 7]
```diff
@@ -1686,212 +1694,215 @@ The old wallet cache file will be renamed and can be restored later.
...
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="46"/>
+        <source>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;&lt;font size=&apos;2&apos;&gt; (&lt;/font&gt;&lt;a href=&apos;#&apos;&gt;Start daemon&lt;/a&gt;&lt;font size=&apos;2&apos;&gt;)&lt;/font&gt;</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="82"/>
+        <source>Ring size: %1</source>
+        <translation>Rengaskoko:</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: Please add `%1` as in the source string

File: translations/monero-core_fi.ts [hunk 8]
```diff
@@ -1686,212 +1694,215 @@ The old wallet cache file will be renamed and can be restored later.
...
+    <message>
+        <location filename="../pages/Transfer.qml" line="429"/>
+        <source>Advanced options</source>
+        <translation>Edistyneet asetukset</translation>
+    </message>
     <message>
         <location filename="../pages/Transfer.qml" line="556"/>
         <source>Can&apos;t load unsigned transaction: </source>
-        <translation type="unfinished"></translation>
+        <translation>Allekirjoittamattoman siirron lataus ei onnistu</translation>
```
Comment (erciccione) [inline translations/monero-core_fi.ts]: Please add `:` + whitespace, as in the source string

**Comment by erciccione** [comment]
Good, thanks. Can you squash those commits in just one? contact me if you need help

**Comment by skftn** [comment]
Nice!

**Pull request merged**